detections/endpoint/detect_password_spray_attack_behavior_from_source.yml

```diff
@@ -0,0 +1,71 @@
+name: Detect Password Spray Attack Behavior From Source
+id: b6391b15-e913-4c2c-8949-9eecc06efacc
+version: 1
+date: '2023-10-30'
+author: Steven Dick
+status: production
+type: TTP
+description: The following analytic identifies one source failing to authenticate with 10 or more unique users. This behavior could represent an adversary performing a Password Spraying attack to obtain initial access or elevate privileges. This logic can be used for real time security monitoring as well as threat hunting exercises and works well against any number of data sources ingested into the CIM datamodel.
+
+Environments can be very different depending on the organization. Test and customize this detections thresholds if needed.

```

Comment:
The linebreak in the above line + this being a new line leads the yaml parser to believe that this is a new key, but there's no `:` or value. Suggest moving it up to be part of the paragraph and this should build properly.

Reply:
Cleaned this up on both of them